test/foundry/NonReentrant.t.sol

```diff
@@ -56,8 +56,8 @@ contract NonReentrantTest is BaseOrderTest {
     }
 
     function testNonReentrant() public {
-        for (uint256 i; i < 7; i++) {
-            for (uint256 j; j < 10; j++) {
+        for (uint256 i; i < 7; ++i) {
```

Comment:
Which was more prevalent post or pre increment? This PR seems to change it at so many places I wonder if the new style was prevalent 😅

Reply:
There is also a claim about gas optimization, I only see the `i++` -> `++i` changes, is that really making any changes for gas?

Reply:
I did update a few `var++ `to `++var` yesterday night, but this morning a bunch of PRs merged main and the gas optimization changes I made were already on master.